data/merge_acs_data.py

```diff
@@ -0,0 +1,130 @@
+import pandas as pd
+import numpy as np
+from pandas.core.reshape.merge import merge
+
+# Read in two files - one, a population by geography file produced by NHGIS
+# The data can be found with these filters:
+# DATASETS : 	2015_2019_ACS5a
+# TOPICS : Total Population
+# The data should be downloaded at the block group level, the highest resolution geo data available from the ACS
+
+# The second file is our master block data set, which is produced elsewhere in this repository.
+# See 'create_master_block_dataset.py'
+
+
+acs_block_groups = pd.read_csv(
+    "original/nhgis/nhgis0002_csv/nhgis0002_ds244_20195_2019_blck_grp.csv",
+    usecols=[
+        "YEAR",
+        "STATEA",
+        "COUNTYA",
+        "TRACTA",
+        "BLKGRPA",
+        "NAME_E",
+        "ALUBE001",
+        "ALUBM001",
+    ],
+)
+
+# low_memory = False is necessary to avoid an error message when loading in the huge block-level data
+
+blocks = pd.read_csv("master_block.csv.gz", low_memory=False,)
+
+
+# Fill missing values that interfere with merge with 0
+
+blocks.fillna(0)
+
+
+# The blockgroup is the same as the first digit of the four digit block number.
+# Therefore, we can obtain it by integer dividing by 1000
+
+blocks["block_group"] = blocks.block // 1000
+
+
+# Here we aggregate our block level data up to the block group level, so that we can then join it with the ACS data,
+# which is also at the block group level
+
+
+merge_list = ["state_fip", "county_fip", "census_tract", "block_group"]
```

Comment:
constant are uppercase in python, "columns" is more precise
```suggestion
MERGE_COLS = ["state_fip", "county_fip", "census_tract", "block_group"]
```